apps/frontend/src/routes/register/index.tsx

```diff
@@ -159,6 +166,21 @@ export default component$(() => {
                     Password must contain at least six characters, including at least 1 letter and 1 number.
                   </span>
                 </label>
+                <div class="flex items-center relative">
+                    <input
+                        name="policies"
+                        type="checkbox"
+                        class="checkbox checkbox-accent"
+                    />
+                     <span class="ml-2 text-xs text-left">
+                        I agree with the <a href="/privacy-policy" class="text-accent">Privacy Policy</a>.
+                    </span>
+                </div>
+                <label class="label">
+                    {action.value?.fieldErrors?.policies &&<span class="label-text text-error text-left">
+                    {action.value?.fieldErrors?.policies}
```

Comment:
You don't need to use the optional chaining here.
```suggestion
                    {action.value.fieldErrors.policies}
```